openapi.yaml

```diff
@@ -1312,6 +1316,7 @@ paths:
         - BearerAuth: []
       tags:
         - idp-group-mappings
+      x-speakeasy-ignore: true
```

Comment:
i dont think this does anything. you usually ignore fields when youre tracking them as part of an entity operation and you dont want the generator to track it. here youre not doing anything with this api (no entity operation annotations)

Reply:
Oh right. I'll remove